addons/maptools/CfgVehicles.hpp

```diff
@@ -52,7 +52,7 @@ class CfgVehicles {
                 };
                 class ACE_MapToolsAlignCompass {
                     displayName = CSTRING(MapToolsAlignCompass);
-                    condition = QUOTE((GVAR(mapTool_Shown) != 0) && {'ItemCompass' in assigneditems ACE_player});
+                    condition = 'GVAR(mapTool_Shown) != 0 && {getUnitLoadout ACE_player param [9, []] param [3, ""] != ""}';
```

Comment:
crashes game on load
```
File z\ace\addons\maptools\CfgVehicles.hpp, line 54: '/CfgVehicles/CAManBase/ACE_SelfActions/ACE_MapTools/ACE_MapToolsAlignCompass.condition': Missing ';' prior '}'
Error context ;
ErrorMessage: File z\ace\addons\maptools\CfgVehicles.hpp, line 54: '/CfgVehicles/CAManBase/ACE_SelfActions/ACE_MapTools.': ''' encountered instead of '='
Application terminated intentionally
ErrorMessage: File z\ace\addons\maptools\CfgVehicles.hpp, line 54: '/CfgVehicles/CAManBase/ACE_SelfActions/ACE_MapTools.': ''' encountered instead of '='
```

```suggestion
                    condition = QUOTE(GVAR(mapTool_Shown) != 0 && {getUnitLoadout ACE_player param [ARR_2(9, [])] param [ARR_2(3, '')] != ''});
```

Reply:
Hm weird, it didn't crash my game when I tried it. 🤔

Reply:
did you do full build?  this was on a filepatching build which might work differently

Reply:
Hm, that might be it.

Reply:
@PabstMirror I'm not sure why that would crash at all, but did it only crash with filepatching?

Reply:
I didn't test with a full (pboProject) build.
But I think i've seen this before, that tool can handle the macros and make a binned config that works, but arma itself can't

Reply:
Just to confirm, it's `condition = 'GVAR(mapTool_Shown) != 0 && {getUnitLoadout ACE_player param [9, []] param [3, ""] != ""}';` that is crashing?

That has fewer macros, and the only macro present is also present in `condition = QUOTE(GVAR(mapTool_Shown) != 0 && {getUnitLoadout ACE_player param [ARR_2(9, [])] param [ARR_2(3, '')] != ''});`